pkg/didcomm/packer/jwe/authcrypt/authcrypt.go

```diff
@@ -46,56 +46,6 @@ type Packer struct {
 	randReader io.Reader
 }
 
-// Envelope represents a JWE envelope as per the Aries Encryption envelope specs
```

Comment:
as discussed @DRK3 , let's not modify existing code in packer/jwe... 

simply use the new JSONWebEncryption you created in this patch for the new JWE documents we're going to use in EDV and the (new) didComm (which will be different than the existing code in packer/jwe/authcrypt)

Reply:
Done